src/utils/daos_hdlr.c

```diff
@@ -2227,12 +2242,13 @@ fs_copy_connect(daos_file_t *daos_src_file,
 			if (rc != 0) {
 				fprintf(stderr, "failed to create container: %s (%d)\n",
 					d_errdesc(rc), rc);
-				return rc;
+				D_GOTO(out, rc);
 			}
 			fprintf(stdout, "Successfully created container "DF_UUIDF"\n",
 				DP_UUID(ap->dst_c_uuid));
 		}
 	}
+out:
```

Comment:
@dsikich In a future iteration, we'll probably want to make sure we handle any necessary cleanup for any errors. For example, if we mount dfs but container creation fails, we need to make sure we unmount the container.

Reply:
sorry, but this should be done in this PR, not a future iteration. we can't have daos leave containers created or cont / dfs handles dangling around. container handles do consumer resources on the server side.

Reply:
@mchaarawi @daltonbohning added the cleanup for these cases